package.json

```diff
@@ -147,13 +147,13 @@
     "node": "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0"
   },
   "resolutions": {
+    "@jest/create-cache-key-function": "workspace:*",
     "@testing-library/dom/pretty-format": "26.6.1",
     "@types/jest/jest-diff": "^25.1.0",
     "@types/jest/pretty-format": "^25.1.0",
     "babel-jest": "workspace:*",
-    "fbjs-scripts": "patch:fbjs-scripts@^1.1.0#./patches/fbjs-scripts.patch",
     "jest": "workspace:*",
     "jest-environment-node": "workspace:*",
-    "react-native": "patch:react-native@0.63.2#./patches/react-native.patch"
+    "react-native": "patch:react-native@0.64.0#./patches/react-native.patch"
```

Comment:
ideally we should get rid of this patch as well, but I haven't bothered creating an issue with RN yet 😅